ResiliencyStorageSnapshotsSample/Readme.md

```diff
@@ -0,0 +1,19 @@
+---
+title: AAF resiliency pillar - Data management - storage resiliency code samples 
+description: Sample scripts for creating storage snapshots for resiliency
+ms.date: 03/19/2020
+author: Magrande
+ms.topic: guide
+ms.service: architecture-framework
+ms.subservice: resiliency
+ms.custom: data-managemt
+---
+
+## Storage resiliency code samples 
```

Comment:
If possible, it would help a lot to have a step by step instruction about how to run each file in the solution and what it is for